<?xml version="1.0"?>
<launch>
    <include file="$(find assignment_2_2023)/launch/sim_w1.launch" />
    <param name="des_pos_x" value= "0.0" />
    <param name="des_pos_y" value= "1.0" />
    <param name="ws_size" value= "10" />
    <node pkg="assignment_2_2023" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="assignment_2_2023" type="go_to_point_service.py" name="go_to_point"  />
    <node pkg="assignment_2_2023" type="bug_as.py" name="bug_action_service" output="screen" />
    <node pkg="assignment_2_2023" type="target_client.py" name="trg_client" launch-prefix="xterm -e" />
    <node pkg="assignment_2_2023" type="LastTrgPos_server.py" name="LastTrgPos_server" output="screen" />
    <node pkg="assignment_2_2023" type="InfoRobot_server.py" name="InfoRobot_server" output="screen" />    
    

</launch>
